src/deprecated/Tribe__Tickets__Google_Event_Data.php

```diff
@@ -0,0 +1,5 @@
+<?php
+_deprecated_file( __FILE__, '4.2', 'Tribe__Events__JSON_LD__Order' );
+
+
+class Tribe__Tickets__Google_Event_Data extends Tribe__Events__JSON_LD__Order {}
```

Comment:
Shouldn't this extend `Tribe__Tickets__JSON_LD__Order` ('Tickets' not 'Events', or is there a matching PR for core)?